docs/sdk/serverless-signing/conditional-signing.md

```diff
@@ -5,123 +5,56 @@ import TabItem from '@theme/TabItem';
 
 # Conditional Signing
 
-## Prerequisites
-
-- Familiarity with JavaScript
-- Basic understanding of [serverless signing](../serverless-signing/quick-start.md)
-
 ## Overview
-Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs. This system can be used to uphold the integrity of data on the open web, in its function as a decentralized notary.
+Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs.
 
 The below example will check if the user has at least 1 Wei on Ethereum, only returning a signature if they do.
 
-:::note
-`toSign` data is required to be in 32 byte format. 
+## Prerequisites
 
-The `ethers.utils.arrayify(ethers.utils.keccak256(...)` can be used to convert the `toSign` data to the correct format.
-:::
+- Knowlege of [SessionSigs](../authentication/session-sigs/intro)
+- Knowledge of how to [generate an AuthSig](../migrations/6.0.0.md#generate-an-authsig)
+- Basic understanding of [serverless signing](../serverless-signing/quick-start)
 
-Installed the latest client on `datil-dev`
+## Complete Code Example
+The complete code example is available in the [Lit Developer Guides Code Repository](https://github.com/LIT-Protocol/developer-guides-code/tree/master/conditional-signing). There is both a browser and Node.js implementation of the code.
 
-```bash
-yarn install @lit-protocol/lit-node-client
-```
+### Example Lit Action
```

Comment:
do we want to add the full code dropdown here? Like wyatt did here: https://developer.litprotocol.com/sdk/serverless-signing/deploying#deploying-using-ipfs

Reply:
I had it in a dropdown at one point, but the full example is 190 lines and multiple files. Combined, the dropdown would be ~230 lines. I think I'd rather have people just view the code example on GitHub

Reply:
ahh yes that is quite large, agreed then